Sources/VRage.Dedicated/Configurator/ConfigForm.cs

```diff
@@ -69,7 +69,7 @@ private void startButton_Click(object sender, EventArgs e)
             else // Local / Console
             {
                 string[] cmdLine = Environment.GetCommandLineArgs();
-                Process.Start(cmdLine[0], ((cmdLine.Length > 1) ? cmdLine[1] : "" ) + " -console -ignorelastsession");
+                Process.Start(cmdLine[0], ((cmdLine.Length > 1) ? String.Join(" ", cmdLine, 1, cmdLine.Length - 1 ) : "") + " -console -ignorelastsession");
```

Comment:
Well if `cmdLine` already contains `-console` it will be part of the `String.Join` and the `+" -console...` will concatenate it a second time. I'm just asking if that is ok for the new process.

And I'm pointing out that there is more than one bug here. So this line will need to be touched again.